requirements/requirements.in

```diff
@@ -10,7 +10,7 @@ cryptography>=36.0.2,<37.0.0 # Until paramiko fixes https://github.com/paramiko/
 Cython<3 # Since the bump to PyYAML 5.4.1 this is now a mandatory dep
 daphne
 distro
-django==3.2.13  # see UPGRADE BLOCKERs
+django>=3.2.16,<4.0.0  # see UPGRADE BLOCKERs https://github.com/ansible/awx/security/dependabot/67
```

Comment:
I'm hesitant to loosen up this pin in this matter.  Sure, due to the requirements.txt it won't be doing hidden upgrades all of the time, but if someone does a pip-compile for some other reason it may bring this along with it.

Reply:
Are you thinking we just want to pin directly to `==3.2.16`?
Or would you be good with just tightening it up to like a minor release like `>=3.2.16,<3.3.0`?

Reply:
Yes, an exact pin (`==3.2.16`)